sale_margin_percentage/views/sale_order_views.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="sale_margin_sale_order" model="ir.ui.view">
+        <field name="name">sale.margin.pct.form.inherit</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale_margin.sale_margin_sale_order"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='margin']" position="attributes">
+                <attribute name="groups">base.group_system</attribute>
+            </xpath>
+            <xpath expr="//field[@name='margin']" position="after">
+                <field name="margin_percentage" class="oe_inline"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="sale_margin_sale_order_line_form" model="ir.ui.view">
+        <field name="name">sale.margin.pct.per.line.form.inherit</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale_margin.sale_margin_sale_order_line_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='purchase_price']" position="after">
```

Comment:
Ambiguous, it could be from the tree view or the form.

Take a look to the xpath just below:

`<xpath expr="//field[@name='order_line']/tree//field[@name='purchase_price']" position="after">`